resources/server_install.rb

```diff
@@ -104,6 +104,11 @@
 property :windows_winrm_timeout, String, default: 'PT60.000S'
 
 action :install do
+  case node['platform_family']
+  when 'debian'
+    apt_update ''
+  end
```

Comment:
you might want this as you only have one case to worry about
```suggestion
apt_update '' if node['platform_family'] == 'debian'
```